### ISSUE TITLE
keyboard+click combo in Firefox to "open in a new tab" links from "cards" opens the link both in origin AND new tab

### ISSUE THREAD

**Issue by koalie** [w3c/w3c-website#297]
**Describe the issue**
In any "card" component, from Firefox on desktop, when I use command+click (the keyboard shortcut to "open in a new tab"), the link opens both in a new tab and in the tab I'm in.

**To reproduce**
For example:
https://beta.w3.org/mission/ go to the very bottom, where "you may be interested in" features cards.
https://beta.w3.org/groups/wg/ where each group is a "card".

On MacOS, I use "command+click" to open in a new tab, and in these cases the page opens twice: in the tab I'm on, and in the new tab.

**Additional context**
Note that when I right-click a link and choose "open in a new tab", the link opens only in the new tab, as expected.

@vivienlacourba has managed to reproduce this fully as well on Firefox on Ubuntu (on Ubuntu "ctrl+left-click" is the keyboard shortcut to open in a new tab), and like on MacOS this behaviour is not triggered when right-clicking and choosing "open in a new tab".

**Comment by nhoizey** [comment]
@koalie on macOS with Firefox, Cmd + click on trackpad opens the link in current tab, not even another tab.

I think there's an open issue in Bugzilla, but I can't find it.

Also, it's a known issue in some JS "enhanced" versions of the inclusive components cards in https://inclusive-components.design/cards/

**Comment by nhoizey** [comment]
Firefox handling of "cmd + (touchpad only?) click" has another issue, maybe related: https://bugzilla.mozilla.org/show_bug.cgi?id=1204440

**Comment by koalie** [comment]
> @koalie on macOS with Firefox, Cmd + click on trackpad opens the link in current tab, not even another tab.

You sure? because https://support.mozilla.org/en-US/kb/mouse-shortcuts-perform-common-tasks states that cmd + (Left-)click Link = Open Link in New Background Tab.

**Comment by koalie** [comment]
> Firefox handling of "cmd + (touchpad only?) click" has another issue, maybe related: https://bugzilla.mozilla.org/show_bug.cgi?id=1204440

" 8 years ago " 😭

**Comment by chriskirknielsen** [comment]
> > @koalie on macOS with Firefox, Cmd + click on trackpad opens the link in current tab, not even another tab.
> 
> You sure? because https://support.mozilla.org/en-US/kb/mouse-shortcuts-perform-common-tasks states that cmd + (Left-)click Link = Open Link in New Background Tab.

I think Nicolas meant that for them it didn't do the doubled tabs when using a cmd+trackpad combo. However I think it may be because Nicolas clicked the card but not on the title itself. (see note below)

I can confirm using Firefox on macOS, I get doubled tabs in all cases:
- Cmd-click via trackpad + built-in keyboard
- Cmd-click via trackpad + external keyboard
- Cmd-click via external mouse + built-in keyboard
- Cmd-click via external mouse + external keyboard

**Note:** This only occurs if clicking the title (which contains the `<a>` element in the card). If clicking the card content, the link is opened in the same tab as if the Cmd key was not pressed during click. (I think this is what your original message alluded to, @koalie, but I just wanted to confirm this was the scenario you were describing)

I initially thought this only happened with external devices, however, I realised it was because I had clicked on the title for the external keyboard/mouse, and on the card content with the trackpad.

**Comment by nhoizey** [comment]
@chriskirknielsen you're right, I didn't get the double tab because I was clicking the card, not its link (I guess that's the point of clickable cards… 😅).

I get the same behavior as @koalie when clicking the link with Cmd + click.

**Comment by japuiglaucia** [comment]
keyboard+click combo in Firefox to "open in a new tab" links from "cards" opens the link both in origin AND new tabs #297
